src/MizarModels/include/MizarModels/SamplingWithFrameTrackReportModel.h

```diff
@@ -79,8 +81,15 @@ class SamplingWithFrameTrackReportModelTmpl : public QAbstractTableModel {
     return kColumnsCount;
   };
   [[nodiscard]] QVariant data(const QModelIndex& index, int role) const override {
-    if (index.model() != this || role != Qt::DisplayRole) return {};
-    return QString::fromStdString(MakeDisplayedString(index));
+    if (index.model() != this) return {};
+    switch (role) {
```

Comment:
Nit: I think some ifs and early returns is a little bit better suited than a switch, since role is not really a closed set. (Feel free to ignore though. No strong opinion here)

Reply:
Even if we go through every value of enum, we still need to add the `default` clause. Apparently, there is not such thing as "closed set".